yarn-project/types/src/event_logs.ts

```diff
@@ -4,7 +4,7 @@ import { randomBytes } from 'crypto';
 /**
  * Data container of unverified data corresponding to one L2 block.
  */
-export class UnverifiedData {
+export class EventLogs {
```

Comment:
Would just rename this to `Logs` and describe in the docs that it's the logs emitted from Noir. Or was the name `Logs` causing some issues? If yes then maybe `NoirLogs` would work.

Reply:
was actually thinking about that, this was the first change I made and didn't like it later on. `Logs` would be a bit confusing I think as we also have `Log` type for Ethereum logs. `NoirLogs` sounds good to me 👍

Reply:
Cool cool, let's go with `NoirLogs` then.